releng/freemarker-ide.tpd

```diff
@@ -0,0 +1,36 @@
+target "Freemarker IDE Target Platform"
+
+with requirements, source, configurePhase
+
+location "https://download.eclipse.org/releases/2021-06" {
```

Comment:
Let's try to target https://download.eclipse.org/releases/2019-06/. Or even older, but 2 years old is generous enough, I think.